task_01.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env/ python
+# -*- coding: utf-8 -*-
+"""A really good docstring for showing the date"""
+
+import datetime
+CURDATE = None
+datetime.date.today()
+
+
+def get_current_date():
+    return CURDATE
```

Comment:
docstrings immediately follow the function name -- your docstring is excellent but the code at L#11 has to start after the docstring.